baybe/serialization/mixin.py

```diff
@@ -3,6 +3,8 @@
 import json
 from typing import Type, TypeVar
 
+import attr
```

Comment:
im not 100% up to date why theres still something called `attr` but for consistency we always should use the `attrs` version of stuff (unless theres a meaningful difference, which doesnt seem the case here)